controllers/storageclassrequest/storageclassrequest_controller_test.go

```diff
@@ -32,26 +33,74 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	pgAutoscaleMode = "pg_autoscale_mode"
+	pgNum           = "pg_num"
+	pgpNum          = "pgp_num"
+	namespaceName   = "test-ns"
+	deviceClass     = "ssd"
+	kind            = "StorageProfile"
```

Comment:
1. Since these consts resides in a function named `getMockStorageProfiles` that's sole purpose is to generate mock `StorageProfile`s, it's pretty derivative.
2. In order to unit test `StorageProfile`s we need to mock up resources that are a replica of the real world resource. Not sure if you're asking what is the usage of those in general or in the unit tests @rewantsoni

Reply:
1. I would still suggest to have it as storageProfileKind 
2. I don't see the usage of the const pgAutoscaleMode, pgNum and pgpNum anywhere in the changes that you made

Reply:
1. I don't see the point, sorry. These consts are accessible only in the scope of the said function that handles mock SPs generation
2.   They are used [here](https://github.com/raaizik/ocs-operator/blob/sp-tests-main/controllers/storageclassrequest/storageclassrequest_controller_test.go#L362-L364)

Reply:
Resolving as per @rewantsoni's manual confirmation